Adds Portable Freezers to the Protolathe designs

ozwe11 opened the pull request:
🆑 
rscadd: Portable freezers now in the Protolathe menu
/:cl:

This adds the portable freezer item to the Protolathe designs.

Requires materials 3, tech 2, and 350 plastic to build.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

ozwe11 committed:
Adds Portable Freezers to the Protolathe designs

🆑 
rscadd: Portable freezers now in the Protolathe menu
/:cl:

This adds the portable freezer item to the Protolathe designs.

Requires materials 3, tech 2, and 350 plastic to build.

HeyBanditoz commented on code/modules/research/designs/designs_singletons.dm:
```diff
@@ -31,6 +31,14 @@
+/datum/design/item/weapon/storage/box/freezer
+	name = "Portable Freezer"
+	desc = "This nifty shock-resistant device will keep your 'groceries' nice and non-spoiled."
+	id= "freezer"
```
```suggestion
	id = "freezer"
```

ozwe11 committed:
Update code/modules/research/designs/designs_singletons.dm

Co-authored-by: Hayden <banditoz@protonmail.com>

Spookerton commented:
@ozwe11 squash dis

ozwe11 committed:
Adds Portable Freezers to the Protolathe designs

🆑 
rscadd: Portable freezers now in the Protolathe menu
/:cl:

This adds the portable freezer item to the Protolathe designs.

Requires materials 3, tech 2, and 350 plastic to build.
Update code/modules/research/designs/designs_singletons.dm

Co-authored-by: Hayden <banditoz@protonmail.com>

ozwe11 committed:
Merge remote-tracking branch 'origin/freezer' into freezer

ozwe11 commented:
closing for now while I sort out a logistics issue
